### PR TITLE
Add unit test for unsupported ANP action

### PR DESCRIPTION
**Pull request by reachjainrahul**
Signed-off-by: Rahul Jain <rahulj@vmware.com>

### CODE REVIEW

File: pkg/controllers/cloud/networkpolicy_test.go [hunk 1]
```diff
@@ -36,6 +36,7 @@ import (
 	logf "sigs.k8s.io/controller-runtime/pkg/log"
 
 	antreanetworking "antrea.io/antrea/pkg/apis/controlplane/v1beta2"
+	"antrea.io/antrea/pkg/apis/crd/v1alpha1"
```
Comment (Anandkumar26) [inline pkg/controllers/cloud/networkpolicy_test.go:39]: nit: antreav1alpha1?

**Review by Anandkumar26 — APPROVED** [review]
/LGTM

File: pkg/controllers/cloud/networkpolicy_test.go (hunk 1 above)
Comment (reachjainrahul) [reply pkg/controllers/cloud/networkpolicy_test.go:39]: its fine

**Pull request merged**